controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java

```diff
@@ -696,49 +696,65 @@ public void listStreamsInScope(Controller.StreamsInScopeRequest request, StreamO
         } else {
             ctx = null;
         }
-
-        authenticateExecuteAndProcessResults(
-                () -> {
-                        String result = this.grpcAuthHelper.checkAuthorization(
-                                AuthResourceRepresentation.ofScope(scopeName),
-                                AuthHandler.Permissions.READ,
-                                ctx);
-                        log.debug("Result of authorization for [{}] and READ permission is: [{}]",
-                            AuthResourceRepresentation.ofScope(scopeName), result);
-                        return result;
-                },
-                delegationToken -> controllerService
-                        .listStreams(scopeName, request.getContinuationToken().getToken(), pageLimit)
+        Function<String, CompletableFuture<Controller.StreamsInScopeResponse>> streamsFn = delegationToken ->
```

Comment:
The AuthContext if..else block could be expressed more concisely using a ternary operator as done in listKeyValueTables :
https://github.com/pravega/pravega/blob/bf961c4e3b39353b9881dbe60252bfbd1fbdae66/controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java#L158